3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -27,9 +27,9 @@ exercise file.
 const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
-  // TODO Refactor this function
   const dice = [1, 2, 3, 4, 5];
-  return rollDie(1);
+  const promisesArray = dice.map((die) => rollDie(die));
```

Comment:
Your arrow function just passes on the `die` argument to `rollDie` and then returns whatever `rollDie` returns. Why not cut out the middleman?

```suggestion
  const promisesArray = dice.map(rollDie);
```